code/modules/research/designs/medical_designs.dm

```diff
@@ -480,6 +480,17 @@
 	build_path = /obj/item/organ/internal/cyberimp/eyes/hud/medical
 	category = list("Medical")
 
+/datum/design/cyberimp_hydroponics_hud
+	name = "Hydroponic HUD Implant"
+	desc = "These cybernetic eye implants will display a hydroponics HUD over everything you see. Wiggle eyes to control."
+	id = "ci-hydrohud"
+	req_tech = list("materials" = 5, "programming" = 4, "biotech" = 4, "magnets" = 3)
+	build_type = PROTOLATHE | MECHFAB
+	construction_time = 50
```

Comment:
```suggestion
	construction_time = 5 SECONDS
```

Reply:
Any reason to change this? I had originally left the construction time at 50 to match the other HUD implants - is this just syntactic sugar?

Reply:
This is more readable, `SECONDS` is a macro that will multiply the number by 10 due to `construction_time` being calculated in deciseconds

Reply:
yes, it's just more readable

Reply:
Very in favor of making the code more readable, so approved. Should I change the other HUD implant construction times to use this macro in this PR or a separate one?